modules/signatures/windows/sniffer_winpcap.py

```diff
@@ -28,7 +28,7 @@ class InstallsWinpcap(Signature):
     mbcs = ["OB0009", "B0023"]
 
     def run(self):
-        indicators = [".*\\\\packet\.dll$", ".*\\\\npf\.sys$", ".*\\\\wpcap\.dll$"]
+        indicators = (r".*\\packet\.dll$", r".*\npf\.sys$", ".*\wpcap\.dll$")
```

Comment:
Should that be `\\wpcap\.dll` not `\wpcap\.dll` ? The file name is `wpcap.dll`.  I am not sure about `\npf\.sys`